metadata.json

```diff
@@ -99,12 +99,12 @@
     },
     {
       "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": "3.5"
```

Comment:
The docs state the following though:

"This command must have a fully qualified path, and should contain a percent (%) token where it would expect an input file. It must exit 0 if the syntax is correct, and non-zero otherwise."

That needs to be addressed before merge.